src/components/stepforms/widgets/FilterSimpleCondition.vue

```diff
@@ -165,6 +173,10 @@ export default class FilterSimpleConditionWidget extends Vue {
   }
 
   get inputWidget(): VueConstructor<Vue> | undefined {
+    if (this.$store.state.vqb.feature_flags.QUERYBUILDER_ESJSON == 'enabled' && this.columnTypes[this.value.column] == 'date') {
+      return InputDateWidget;
+    }
+
```

Comment:
Ultimately, I thnik we should have a 2 dimension array (operator, and column type) to pick a widget

Reply:
+1 When filtering with the UI, directly from the column contextual menu, often multiple values will be selected.
For these, we use a MultiInputText, which outputs an array of values, represented by tags.
Maybe we should keep this with more priority than the input type="date" for in and not in operators?